### PR TITLE
List of default web hooks could be setted up

### PR DESCRIPTION
**Pull request by newmen**
Now we can setup our GitLab for auto create web hooks for each new project

### CODE REVIEW

File: config/gitlab.yml.example [hunk 1]
```diff
@@ -80,6 +80,11 @@ production: &base
       wiki: true
       snippets: false
 
+      ## Default web hooks settings
```
Comment (jvanbaarsen) [inline config/gitlab.yml.example]: @randx We are trying to move all the settings to the settings page right? So no new settings should be added to gitlab.yml?

File: app/models/project.rb [hunk 2]
```diff
@@ -56,6 +56,14 @@ def set_last_activity_at
     update_column(:last_activity_at, self.created_at)
   end
 
+  # create default hooks if they are presented
+  after_create :create_default_web_hooks, if: -> { !!gitlab_config_features.auto_web_hooks }
```
Comment (jvanbaarsen) [inline app/models/project.rb]: Can you please move the after_create to the top of the file? That way we can see in one look what callbacks are being called. Also the comment seems to be redundant, you can remove that one

File: config/gitlab.yml.example (hunk 1 above)
Comment (randx) [inline config/gitlab.yml.example]: @jvanbaarsen right. Except cases like path to `git` binary

Comment (jvanbaarsen) [inline config/gitlab.yml.example]: Ok, so for this PR the config needs to be moved to some web interface right? Can you say if you would like to see this merged in? If so I'll guide the rest of the PR.

**Comment by dblessing** [comment]
@newmen Can you please change this to put the setting in the admin section of the UI?

**Comment by jvanbaarsen** [comment]
This merge request has been closed because a request for more information has not been reacted to for more than 2 weeks. If you respond and conform to the merge request guidelines in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#pull-requests) we will reopen this merge request.

**Pull request closed without merging**